environment-dev.yml

```diff
@@ -14,8 +14,8 @@ dependencies:
   - importlib_metadata  # drop when pycalphad drops support for Python<3.8
   - matplotlib-base>=3.3
   - numpy>=1.13
-  - pyparsing
-  - python>=3.6
+  - pyparsing>=2,<3
```

Comment:
This is fine, and it's not a blocker for 0.9, but please open an issue to track pyparsing 3 errors.

Reply:
Done in #347

Reply:
@bocklund Thanks for opening #347